pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "rustbininfo"
-version = "0.5.3"
+version = "0.5.4"
```

Comment:
Could you just remove this part ? I will be doing the version upgrades myself :)

Reply:
Sure, it is removed.